worlds/factorio/data/mod_template/control.lua

```diff
@@ -310,7 +310,12 @@ script.on_event(defines.events.on_player_removed, on_player_removed)
 
 function on_rocket_launched(event)
     if event.rocket and event.rocket.valid and storage.forcedata[event.rocket.force.name]['victory'] == 0 then
-        if event.rocket.get_item_count("satellite") > 0 or GOAL == 0 then
+		satellite_count = 0
+		cargo_pod = event.rocket.cargo_pod
+		if cargo_pod then
+			satellite_count = cargo_pod.get_item_count("satellite")
+		end
+        if satellite_count > 0 or GOAL == 0 then
```

Comment:
Indentation looks weird, but code otherwise looks correct

Reply:
Was a tabs vs space issue.  Resolved it with another commit.